src/matter/cluster/server/OperationalCredentialsServer.ts

```diff
@@ -58,7 +64,25 @@ export const OperationalCredentialsClusterHandler: (conf: OperationalCredentials
 
         // TODO: create ACL with caseAdminNode
 
-        return {status: Status.Success};
+        return {status: OperationalCertStatus.Success};
+    },
+
+    updateOperationalCert: async ({ request: {operationalCert, intermediateCaCert, }, session}) => {
+        // TODO add logic
```

Comment:
throw new Error("Not implemented");

This will help debug if ever this case is triggered since otherwise it will fail later in the flow.
(same below)